Content.Client/HealthAnalyzer/UI/HealthAnalyzerBoundUserInterface.cs

```diff
@@ -33,7 +33,7 @@ protected override void ReceiveMessage(BoundUserInterfaceMessage message)
             if (message is not HealthAnalyzerScannedUserMessage cast)
                 return;
 
-            _window.Populate(cast);
+            _window.UpdateState(new HealthAnalyzerUiState(cast));
```

Comment:
The `HealthAnalyzerSystem` should just send a `HealthAnalyzerUiState`

Reply:
Can I do that if `HealthAnalyzerUiState` extends `BoundUserInterfaceState` and not `BoundUserInterfaceMessage`?

Reply:
yes